tests/conftest.py

```diff
@@ -179,6 +182,36 @@ def _setup_local_spark(out: TerminalReporter, verbosity=0):
 def api_version(request):
     return request.param
 
+# TODO: Deduplicate code with openeo-python-driver
```

Comment:
@soxofaan Is there an easy way to share test helpers? It is not very common to make test code available from other repositories as far as i know

Reply:
that _Sleeper implementation actually comes from python client I think. There I started putting testing utilities  under the `openeo.testing` namespace: https://github.com/Open-EO/openeo-python-client/tree/master/openeo/testing

We also have that with python driver: https://github.com/Open-EO/openeo-python-driver/blob/master/openeo_driver/testing.py

Reply:
https://github.com/Open-EO/openeo-geotrellis-extensions/issues/329#issuecomment-2432635380